libs/hdf-converters/src/prisma-mapper.ts

```diff
@@ -0,0 +1,181 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {
+  BaseConverter,
+  ILookupPath,
+  MappedTransform,
+  parseCsv
+} from './base-converter';
+
+export type PrismaControls = {
+  records: Record<string, string>[];
+};
+// There is a Common Vulnerability Scoring System (cvss) field that contains a cvss qualitative severity rating,
+// however this field does not correlate to the "Severity" (low, moderate, important, high, and critical) field
+// that is mapped to the hdf "impact" field. Severity is mapped to the SEVERITY_LOOKUP"
+const SEVERITY_LOOKUP: Record<string, number> = {
+  low: 0.3,
+  moderate: 0.5,
+  important: 0.7,
+  high: 0.9,
+  critial: 1
+};
+
+function getTitle(item: Record<string, string>): string {
+  const hostName = _.get(item, 'Hostname');
+  const distro = _.get(item, 'Distro');
+  const type = _.get(item, 'Type');
+  return hostName.toString().concat(distro, type);
+}
+
+export class PrismaControlMapper extends BaseConverter {
+  data: PrismaControls = {
+    records: []
+  };
+  mappings: MappedTransform<ExecJSON.Execution, ILookupPath> = {
+    platform: {
+      name: 'Heimdall Tools',
+      release: HeimdallToolsVersion,
+      target_id: 'Prisma Cloud Scan Report'
+    },
+    version: HeimdallToolsVersion,
+    statistics: {
+      duration: null
+    },
+    profiles: [
+      {
+        name: 'Palo Alto Prisma Cloud Tool',
+        version: HeimdallToolsVersion,
+        title: 'Prisma Cloud Scan Report',
+        maintainer: null,
+        summary: '',
+        license: null,
+        copyright: null,
+        copyright_email: null,
+        supports: [],
+        attributes: [],
+        depends: [],
+        groups: [],
+        status: 'loaded',
+        controls: [
+          {
+            path: 'records',
+            key: 'id',
+            tags: {
+              cve: {path: 'CVE ID'},
+              cvss: {path: 'cssv'}
+            },
+            descriptions: [],
```

Comment:
might need to do conditional processing of the 'Description' in order to extract stuff like 'Mitigation' (analogous to fix text) and put it in here.  i think @ejaronne would be pretty interested in going through the possible variations that can exist within this field